geoips/commandline/log_setup.py

```diff
@@ -16,6 +16,21 @@
 import sys
 
 
+def log_with_emphasis(LOG, message):
+    """Log the given message with emphasis provided a certain log type.
+
+    Parameters
+    ----------
+    LOG: logging function
+        Can be of any type [debug, info, interactive...]
+    message: str
+        The message to be logged with emphasis
+    """
+    LOG("    " + "*" * (len(message) + 8))
+    LOG("    " + "*** " + message + " ***")
+    LOG("    " + "*" * (len(message) + 8))
```

Comment:
I think we need to pass in a list of strings, and just put one set of asterisks before the block of lines, and another set after.  1.12.0 will have something along these lines coming - so can probably hold off on finishing this until 1.12 is released ?

We probably also just want to have a fixed number of asterisks (not dependent on the length of the message, especially since it can be multiple lines inside the asterisk block)

Reply:
@evrose54 Can you address Mindy's comment here? I like having the number of asterisks be dynamic, but maybe there should be a maximum length and automatically split `message` into multiple lines to fit within that length?

Reply:
This has been fixed.